### PR TITLE
Fix flush logtail

### PR DESCRIPTION
**Pull request by aptend**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #6753

## What this PR does / why we need it:
- logtail do not read from s3, back to sending meta
- if logtail size is large, force flush table and try again

### CODE REVIEW

**Review by XuPeng-SH — APPROVED** [review]
lgtm

File: pkg/vm/engine/tae/db/db_test.go [hunk 1]
```diff
@@ -3216,6 +3216,8 @@ func (c *dummyCpkGetter) CollectCheckpointsInRange(start, end types.TS) (string,
 	return "", types.TS{}
 }
 
+func (c *dummyCpkGetter) FlushTable(dbID, tableID uint64, ts types.TS) error { return nil }
```
Comment (fengttt) [inline pkg/vm/engine/tae/db/db_test.go:3219]: Wait -- how does this help testing?

**Review by fengttt — COMMENTED** [review]
Can you write a bvt test on this?

Idea is simple, we have a fault injection function.   You can inject a fault in DN, such that even log tail is small, a poll on log tail still flushes it, and then retry.

**Comment by aptend** [comment]
> Can you write a bvt test on this?
> 
> Idea is simple, we have a fault injection function. You can inject a fault in DN, such that even log tail is small, a poll on log tail still flushes it, and then retry.

ok, I'll take a look

**Comment by aptend** [comment]
> > Can you write a bvt test on this?
> > Idea is simple, we have a fault injection function. You can inject a fault in DN, such that even log tail is small, a poll on log tail still flushes it, and then retry.
> 
> ok, I'll take a look

I add in next pr

**Pull request merged**